app/controllers/servers_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class ServersController < ApplicationController
   before_action :set_server, only: %i[show edit update destroy]
-  before_action :authenticate_employee, only: %i[show]
+  before_action :authenticate_employee
   before_action :authenticate_admin, only: %i[new create edit update destroy]
   # GET /servers
   # GET /servers.json
```

Comment:
I think that was correct before.
An employee may see all servers, but may not create a new one or edit existing ones. Only admins are allowed to do so.

Reply:
What Bjoern says is correct.